app/views/questions/widowed_year/edit.html.erb

```diff
@@ -2,8 +2,6 @@
 <% content_for :form_card do %>
   <%= form_with model: @form, url: current_path, local: true, method: "put", builder: VitaMinFormBuilder, html: {class: "form-card"} do |f| %>
     <%= f.vita_min_select(:widowed_year, content_for(:form_question), year_options_from_2014_to_current_tax_yr_plus_one) %>
-    <button class="button button--primary button--wide" type="submit">
-      <%=t("general.next") %>
-    </button>
+    <%= f.continue %>
```

Comment:
We called Anu to ask why some continue buttons say `t("general.next")`. She said (to paragraphase), "Oh no! They should say continue." So here (and a few other places) we got rid of `general.next`.